tests/e2e/e2e_setup_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	b64 "encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/cosmos/cosmos-sdk/x/group"
```

Comment:
Should be moved down with the other sdk incl.

Reply:
please sort the imports